### PR TITLE
Add permissions policy check for FedCM

### PR DESCRIPTION
**Pull request by johannhof**
Companion change to https://github.com/fedidcg/FedCM/pull/236

***
<a href="https://pr-preview.s3.amazonaws.com/johannhof/webappsec-credential-management/pull/204.html" title="Last updated on Aug 29, 2022, 10:05 PM UTC (e77001f)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webappsec-credential-management/204/8350696...johannhof:e77001f.html" title="Last updated on Aug 29, 2022, 10:05 PM UTC (e77001f)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -947,6 +950,12 @@ spec:css-syntax-3;
     1.  Let |sameOriginWithAncestors| be `true` if |settings| is [=same-origin with its
         ancestors=], and `false` otherwise.
 
+    1.  If |options|[{{CredentialRequestOptions/identity}}] [=map/exists=] then
+        if |settings|' [=relevant global object=]'s [=associated Document=] is **not**
+        [=allowed to use=] the "fedcm"
```
Comment (nsatragno) [inline index.bs:955]: Can we link the "fedcm" feature policy to a definition on the fedcm spec?

**Review by nsatragno — APPROVED** [review]
Approved with one optional comment.

File: index.bs (hunk 1 above)
Comment (npm1) [inline index.bs:955]: Per my comment on the other PR, let's rename the value to something different from `fedcm`.

Comment (johannhof) [reply index.bs:955]: Now that we've settled on a new name on the other PR, I'll update this one as well.

**Review by nsatragno — APPROVED** [review]
Approved, happy to merge if @npm1 is happy with it.

File: index.bs [hunk 2]
```diff
@@ -72,8 +72,13 @@ spec: web-otp; urlPrefix: https://wicg.github.io/web-otp
   type: interface
     text: OTPCredential; url: otpcredential
 spec: FEDCM; urlPrefix: https://fedidcg.github.io/FedCM
+  type: dfn
+    text: identity-credentials-get
   type: interface
     text: IdentityCredential; url: identitycredential
+  type: dict-member
```
Comment (npm1) [inline index.bs:79]: I guess we should export these from FedCM so they can automagically be used here... but this is fine for now.

File: index.bs [hunk 3]
```diff
@@ -947,6 +952,12 @@ spec:css-syntax-3;
     1.  Let |sameOriginWithAncestors| be `true` if |settings| is [=same-origin with its
         ancestors=], and `false` otherwise.
 
+    1.  If |options|[{{CredentialRequestOptions/identity}}] [=map/exists=] then
```
Comment (npm1) [inline index.bs:955]:
```suggestion
    1.  If |options|[{{CredentialRequestOptions/identity}}] [=map/exists=] and
```

**Review by npm1 — APPROVED** [review]

File: index.bs (hunk 2 above)
Comment (johannhof) [reply index.bs:79]: Yeah, I'm curious why they don't get picked up, I tried exporting the PP name. The spec works for now, but I'll ask someone wiser than me what we're doing wrong here..

File: index.bs (hunk 3 above)
Comment (johannhof) [reply index.bs:955]: Hm, so, I was trying to stay consistent with the wording below. I'm not sure either is necessarily idiomatic but then I don't think either is wrong. Thoughts?

Comment (nsatragno) [reply index.bs:955]: Roll the dice and pick one, apply to both :)

slight preference for "and" since we're talking about this

Comment (johannhof) [reply index.bs:955]: and it is, then!

File: index.bs (hunk 2 above)
Comment (johannhof) [reply index.bs:79]: Talked to wise sage @tabatkins who will add FedCM to the spec database for us(in a grand ceremony, I expect). That should fix this.

Thank you, Tab!

Comment (tabatkins) [reply index.bs:79]: Added now, give the database an hour or so for the next run to pick up the definitions.

**Pull request merged**